vnext/Shared/DevSupportManager.cpp

```diff
@@ -232,6 +237,30 @@ void DevSupportManager::StopPollingLiveReload() {
   m_cancellation_token = true;
 }
 
+void DevSupportManager::startInspector(const std::string &packagerHost, const uint16_t packagerPort) {
+#ifdef USE_HERMES
+  std::string packageName("RNW");
+  if (winrt::Windows::ApplicationModel::Package::Current())
+    std::string packageName = winrt::to_string(winrt::Windows::ApplicationModel::Package::Current().DisplayName());
+
+  std::string deviceName("RNWHost");
+  auto hostNames = winrt::Windows::Networking::Connectivity::NetworkInformation::GetHostNames();
```

Comment:
does this require new Capabilities declarations on the appx?

Reply:
Not sure !

Reply:
In the current state, these values can be anything.. 

On Android, there is code which relies on these values : https://github.com/facebook/react-native/blob/ca499a61971de30695b7834fb5a16e09bcfb6074/ReactCommon/hermes/inspector/chrome/AutoAttachUtils.cpp#L40

Essentially, on POSIX system there is a special way to enable "pause on load" .. On Windows, we don't have that.. On windows, the "pause on load" will be enabled on refresh.